anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/DetectorLookup.java

```diff
@@ -45,6 +46,10 @@ public DetectorLookup() {
         detectorMap.put("pewma-detector", PewmaAnomalyDetector.class);
     }
     
+    public Set<String> getDetectorTypes() {
```

Comment:
I'd like to see a Unit Test for this class.
Can we use reflection to pick up the list of classes?